core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -559,7 +560,7 @@ class ContainerPool(
             try {
                 info(this, s"making new container because none available")
                 startingCounter.next()
-                makeGeneralContainer(key, containerName, imageName, limits, action.exec.pull)
+                makeGeneralContainer(key, containerName, imageName, limits, action.exec.asInstanceOf[CodeExec[_]].pull)
```

Comment:
That seems bold? `makeWhiskContainer` doesn't get called for non-JS actions? If so can we add a `require` at the top? (or at least a comment similar to the other new ones)